js/entry-points/tracing.js

```diff
@@ -140,6 +140,14 @@ audion.entryPoints.enumAudioBufferProperties_ =
 
 
 /**
+ * If set to true, Web Audio Inspector no longer tracks web audio calls so that
+ * AudioNodes can be GC-ed. This is desired when the user is not using developer
+ * tools.
+ */
+audion.entryPoints.audioUpdatesAreMissing_ = false;
```

Comment:
IIUC, this doesn't trigger anything. This only prevents Audion from collecting nodes in the future. Where are we dumping the data already collected?